README.md

```diff
@@ -1,5 +1,3 @@
-# terraform-module-acme-function
-Automatic Certificate Management
 # https://github.com/hmcts/terraform-module-acme-function
 
 Terraform module to create a Function App which automates certificate management.
```

Comment:
this should mention https://github.com/shibayan/keyvault-acmebot and lets encrypt